### ISSUE TITLE
On each Deploy, Log: `approval not found: record not found`

### ISSUE THREAD

**Issue by mahaffey** [keel-hq/keel#466]
I am using ECR with polling at 1m intervals with the kubernetes provider

I am not using approvals, yet I get this log message with each update: 

```
ime="2019-12-18T01:26:40Z" level=warning msg="provider.kubernetes: got error while archiving approvals counter after successful update" error="approval not found: record not found" kind=deployment name=backendnamespace=default
time="2019-12-18T01:26:40Z" level=info msg="provider.kubernetes: resource updated" kind=deployment name=backendnamespace=default new=2.105.1 previous=2.105.0
```

Nothing is affected really, other than just seeing these logs.

**Comment by Anishmourya** [comment]
@mahaffey don't need to pass AWS secrets in keel .its not working , i removed AWS key and secretes in keel deployment and used kube Imagepullsecrets that worked for me for CI/CD  .
You can define in values.yaml to setup keel

<pre>
keel:
  # keel policy (all/major/minor/patch/force)
  policy: force
  trigger: poll
  pollSchedule: "@every 1m"
  matchTag: true
  # images to track and update
  images:
    - repository: image.repository
      tag: image.tag
      imagePullSecret: ecr-docker-secret
</pre>
Note:- ecr-docker-secret is imagepullsecrets that expire in every 12 hours.
you can use "https://github.com/engineering-bjs/ecr-k8s-secret-creator" to refresh token .

**Comment by Anishmourya** [comment]
@mahaffey earlier was thinking its throwing error due to ECR issue but its not . i have looked into code as well .Its just warning message , you can ignore this . check this function which calls when keel finish update job.
<pre>
// updateComplete is called after we successfully update resource
func (p *Provider) updateComplete(plan *UpdatePlan) error {
	return p.approvalManager.Archive(getIdentifier(plan.Namespace, plan.Name, plan.NewVersion))
}
</pre>
basically it trying to get value of approval but that does not have value in sqlite db . It does set when you pass "approvals: 1" in keel config , then isApproved() function  calls approvalmanger's create() function which set value to sqlite db.

**Comment by plpetkov-tech** [comment]
This issue is not related to ECR, wherever you have the k8s cluster and wherever you pull images from you get this logs filling the logs with garbage

**Comment by mahaffey** [comment]
Haven't used this product in years since my last company so closing the issue
